csbuild/toolchain/__init__.py

```diff
@@ -208,7 +207,7 @@ class CompileChecker(object):
 	Class to implement various components of checking whether a file should be recompiled.
 	"""
 	def __init__(self):
-		self.memo = memo.Memo()
+		self.memo = {} # memo.Memo()
```

Comment:
Since the Memo type no longer exists, does it need to remain here in the comment?